include/tvm/relay/pass.h

```diff
@@ -296,6 +296,17 @@ struct StructuralHash {
   size_t operator()(const Expr& expr) const;
 };
 
+/*! \brief Remove let binding and directly share via pointer instead.
+ *
+ * It will remove all let binding,
+ * and turn all of the variable bound by let into direct pointer reference.
+ *
+ * \param e the expression.
+ *
+ * \return the expression in graph normal form.
+ */
+Expr ToGNF(const Expr& e);
```

Comment:
GNF is not a common term, perhaps ToGraphNormalForm?

Reply:
+1 for ToGraphNormalForm: please use descriptive names so that others that come from a different angle can read the source code.